src/site/layouts/vet_center.drupal.liquid

```diff
@@ -131,10 +131,22 @@
                 typeOfCare = "other"
             %}
 
-            {% if fieldCcVetCenterFaqs.fetched.field_section_header.0.value != empty %}
-                <h2 id="how-different-than-clinic" class="vads-u-margin-top--0 vads-u-font-size--lg
-          small-screen:vads-u-font-size--xl vads-u-margin-bottom--2"> {{ fieldCcVetCenterFaqs.fetched.field_section_header.0.value }}</h2>
-                <p> {{ fieldCcVetCenterFaqs.fetched.field_section_intro.0.value }} </p>
+            {% if fieldCcVetCenterFaqs.fetched %}
+                {% if fieldCcVetCenterFaqs.fetched.field_section_header.0.value != empty %}
+                    <h2 id="how-different-than-clinic" class="vads-u-margin-top--0 vads-u-font-size--lg
+          small-screen:vads-u-font-size--xl vads-u-margin-bottom--2"> {{ fieldCcVetCenterFaqs.fetched.field_section_header.0.value }}
+                    </h2>
+                    <p> {{ fieldCcVetCenterFaqs.fetched.field_section_intro.0.value }} </p>
+                {% endif %}
+                <div class="vads-u-margin-bottom--3" id="field-vet-center-faqs">
+                    {% for q in fieldCcVetCenterFaqs.fetched.field_questions %}
+                        <va-accordion bordered id="cc-question-accordion-{{ q.target_id }}" >
+                            <va-accordion-item id="cc-question-accordion-item-{{ q.target_id }}" header="{{ q.field_question.0.value }}" level="3">
+                                {{ q.field_answer.0.field_wysiwyg.0.processed | drupalToVaPath | outputLinks }}
+                            </va-accordion-item>
+                        </va-accordion>
+                    {% endfor %}
+                </div>
```

Comment:
Is there any way we can use `src/site/paragraphs/q_a.drupal.liquid` here instead? I thought we wanted the centralized content to be dynamic.

Reply:
it's not the same structure, we can't just re-use it. I prefer we maintain our own accordions

Reply:
<img width="1169" alt="Screen Shot 2021-05-06 at 5 33 01 PM" src="https://user-images.githubusercontent.com/100468/117373581-20e06580-ae91-11eb-958c-35d513bbaa50.png">

Reply:
The accordions are in `src/site/paragraphs/q_a.collapsible_panel.drupal.liquid`.

We're massaging the data anyway, so it seems we could make it work with the existing templates, which would have the advantage of consistency across the site and less work on our end to maintain separate templates.

Let's discuss further tomorrow.

Reply:
For the record - we decided to use our own accordion component since we want to use `va-accordion`, and we didn't want to risk breaking existing pages.